README.md

```diff
@@ -74,7 +74,7 @@ Copy `hazard_workflow_input_example.yml`, rename to `hazard_workflow_input.yml`,
 You can then invoke the CWL with:
 
 ```
-$ cwltool hazard_workflow.cwl hazard_workflow_input.yml
+$ cwltool hazard_workflow.cwl#produce-indicator hazard_workflow_input.yml
```

Comment:
Should be `#produce-hazard-indicator` - or should be changed in the CWL file.

Reply:
Woops! Fixed.